src/DynamoSandbox/DynamoCoreSetup.cs

```diff
@@ -76,7 +76,20 @@ public void RunApplication(Application app)
                 Dynamo.Applications.StartupUtils.ASMPreloadFailure -= ASMPreloadFailureHandler;
 
             }
+            catch(FileLoadException e)
+            {
+                DynamoModel.IsCrashing = true;
```

Comment:
why set this?

Reply:
I checked, this is used only for analytics (AFAIK) and since I'm not reporting to analytics here, I can remove this.